libc/include/ctype.h.def

```diff
@@ -10,6 +10,7 @@
 #define LLVM_LIBC_CTYPE_H
 
 #include "__llvm-libc-common.h"
+#include "llvm-libc-types/locale_t.h"
```

Comment:
please add a todo: change this so that it's included only on relevant systems (and same on the other headers now including locale types that aren't `locale.h`